services/books.js

```diff
@@ -3,6 +3,7 @@
 var memoize = require('memoizeasync'),
     db = require('./db'),
     logger = require('./logger'),
+    Console = require('console'),
```

Comment:
I thought so, but wasn't getting anything in the console, I will remove
that.

On Wed, Dec 9, 2015, 3:53 PM Guido García <notifications@github.com> wrote:

> In services/books.js
> <https://github.com/books4equality/books4equality/pull/42#discussion_r47150040>
> :
>
> > @@ -3,6 +3,7 @@
> >  var memoize = require('memoizeasync'),
> >      db = require('./db'),
> >      logger = require('./logger'),
> > +    Console = require('console'),
>
> No need to require console. You can always call console.log("something")
> in javascript.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/books4equality/books4equality/pull/42/files#r47150040>
> .
>